sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -780,7 +780,9 @@ def _add_argparse_args(cls, parser):
 
 
 class PortableOptions(PipelineOptions):
-
+  """Portable options are common options expected to be understood by most of
```

Comment:
Fine with me to remove RunnerOptions because it is just a stub. Not sure about PortableOptions => PortableRunnerOptions because we currently do not include Runner in any option name, e.g. `FlinkPipelineOptions`, not `FlinkRunnerPipelineOptions`.

Reply:
I agree that we can remove the RunnerOptions as its not used anywhere.
I would like to stick with the current convention of not adding "Runner" to PortableOptions.

Reply:
I think PortableOptions is understandable by developers but might be less user-friendly, since users might not know what portability refers to.